### PR TITLE
feature/create_post

### PR DESCRIPTION
**Pull request by mustafa-sab3alarab**

### CODE REVIEW

**Review by AmeerAmjed — APPROVED** [review]
Good job

File: src/main/kotlin/com/the_chance/data/post/Post.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.the_chance.data.post
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Post(
+    val id: String,
+    val content: String,
+    val createdAt: String
```
Comment (Karrar-Mohammed) [inline src/main/kotlin/com/the_chance/data/post/Post.kt:9]: this need to be long

File: src/main/kotlin/com/the_chance/endpoints/Posts.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+
+fun Routing.postsRoutes(postService: PostService) {
+
+    post("/post") {
+        val params = call.receiveParameters()
+        val content = params["content"]?.trim().orEmpty()
```
Comment (Karrar-Mohammed) [inline src/main/kotlin/com/the_chance/endpoints/Posts.kt:15]: content should not be null

**Review by Karrar-Mohammed — APPROVED** [review]
well done 👏

**Pull request merged**